src/sentry/static/sentry/app/components/forms/selectControlLegacy.tsx

```diff
@@ -88,6 +91,13 @@ class SelectControlLegacy extends React.Component<Props> {
         typeof choices === 'function' ? choices(this.props) : choices
       ) || options;
 
+    const isMulti = props.multiple || props.multi;
+
+    // HACK(mgaeta): ReactSelect isn't reading `default` correctly so we're using `value`.
```

Comment:
@mgaeta I think this is because when you use a form model, you need to set the initial state that way instead of by passing a prop to the select component